### PR TITLE
Redesigned conflicts dialog

### LINKED ISSUE #5645: Improving the conflict dialog UI
image

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
- Closes #5645

**Details of Changes**
- Redesigned conflicts dialog. Currently wip.

**Validation**
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

File: src/Files.Uwp/Dialogs/FilesystemOperationDialog.xaml [hunk 1]
```diff
@@ -52,6 +52,7 @@
 
         <vc:ImageModelToImageConverter x:Key="ImageModelToImageConverter" />
         <vc:GenericEnumConverter x:Key="GenericEnumConverter" />
+        <vc:BoolInvertConverter x:Key="BoolInvertConverter" />
```
Comment (gave92) [inline src/Files.Uwp/Dialogs/FilesystemOperationDialog.xaml:55]: Toolkit includes a BoolNegationConverter:
```xaml
xmlns:converters="using:Microsoft.Toolkit.Uwp.UI.Converters"
...
<converters:BoolNegationConverter x:Key="BoolNegationConverter" />
```

**Comment by d2dyno1** [comment]
I think most of the stuff is done,. Delete dialog UI needs to be improved :)

**Review by yair100 — DISMISSED** [review]
This is a great improvement!

**Comment by gave92** [comment]
Looks good 🚀. @yair100 do we want to show the conflicts dialog when copy/pasting to the same folder? I remember you wanted to do this once we had the new dialog.

**Comment by yair100** [comment]
> Looks good 🚀. @yair100 do we want to show the conflicts dialog when copy/pasting to the same folder? I remember you wanted to do this once we had the new dialog.

I've gotten used to the new behavior so I think we can keep it as is.

**Review by yair100 — APPROVED** [review]

**Comment by gave92** [comment]
Just a note: this brings back issue #5788 (in the conflicts dialog the focus should be on the primary button on open):

**Comment by yaichenbaum** [comment]
@d2dyno1 can you fix this?

**Review by gave92 — APPROVED** [review]

**Comment by haywirephoenix** [comment]
Is it possible to change the default choice option?

**Comment by earshinov** [comment]
@haywirephoenix , I just filed a new issue to track that. I am also interested.

**Pull request merged**